packages/extract/src/Extract.js

```diff
@@ -239,4 +241,31 @@ export default class Extract
         this.renderer.extract = null;
         this.renderer = null;
     }
+
+    /**
+     * Takes premultiplied pixel data and produces regular pixel data
+     *
+     * @param pixels {number[] | Uint8Array | Uint8ClampedArray} array of pixel data
```

Comment:
One small change, could we make this private? I don’t like exposing new APIs we have to maintain backward compatibility for if it’s not absolutely necessary. If someone asks, we can make it public or move to utils or something.

Reply:
Agree